cellxgene_resources/upload_local.ipynb

```diff
@@ -26,8 +26,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "scc_repo_loc = os.path.expanduser('~/GitHub/CZI/')\n",
-    "api_key_file_path = os.path.expanduser('~/Documents/cxg-api-key-dev.txt')"
+    "scc_repo_loc = os.path.expanduser('~/GitClones/Lattice-Data/')\n",
```

Comment:
I think this should be `~/GitClones/CZI` since it is the single-cell-curation repo from the CZI organization, not one of ours